cloud_governance/common/aws/utils/utils.py

```diff
@@ -10,8 +11,13 @@ class Utils:
     This is global methods
     """
 
-    def __init__(self):
-        pass
+    def __init__(self, region: str = 'us-east-2'):
+        self.region = region
+        self.ec2_client = boto3.client('ec2', region_name=region)
+        self.iam_client = boto3.client('iam')
+        self.s3_client = boto3.client('s3')
+        self.elb_client = boto3.client('elb', region_name=region)
+        self.elbv2_client = boto3.client('elbv2', region_name=region)
 
```

Comment:
Why we need all the the client init here ?

Reply:
to get the load_balancesrs, users,  roles, and s3. All are calling from here only.

Reply:
Ok got it

Reply:
Sorry, I forgot to delete it here.